src/service/resources-impl.js

```diff
@@ -732,10 +740,12 @@ export class Resources {
         const iniBox = resource.getInitialLayoutBox();
         const diff = request.newHeight - box.height;
         if (diff == 0) {
-          // Nothing to do.
+          // Nothing to resize, resolve attemptChangeSize promise.
+          if (request.callback) {
```

Comment:
We [decided](https://github.com/ampproject/amphtml/pull/4015/files#r71392390) the other noops were rejections.